File: Demo-LLP-MIP/Demo-LLP-MIP-Python/utils/LLianPayDateUtils.py
from datetime import datetime

class LLianPayDateUtils:
    DefaultPattern = "%Y%m%d%H%M%S"

    def __init__(self):
        pass

    @staticmethod
    def get_timestamp():
        return LLianPayDateUtils.format(datetime.now(), LLianPayDateUtils.DefaultPattern)

    @staticmethod
    def format(date, pattern):
        return date.strftime(pattern)

File: Demo-LLP-MIP/Demo-LLP-MIP-PHP/utils/LLianPayDateUtils.php
<?php
class LLianPayDateUtils {
    const DefaultPattern = "YmdHis";

    public function __construct() {}

    public static function get_timestamp() {
        $shanghai_tz = new DateTimeZone('Asia/Shanghai');
        $now = new DateTime('now', $shanghai_tz);
        return LLianPayDateUtils::format($now, LLianPayDateUtils::DefaultPattern);
    }

    public static function format($date, $pattern) {
        return $date->format($pattern);
    }
}


?>